internal/ratelimit/ratelimit.go

```diff
@@ -73,13 +74,21 @@ func NewThrottle(rps int, opts ...Option) *Throttle {
 		options.clock = clock.NewReal()
 	}
 
+	if rps <= 0 {
```

Comment:
I think I will leave it as is. Internally it’s casting into an int64, but I already made the choice to use a plain int at the interface for simplicity. Folks sometimes do math with these numbers, which is just easier to do when all terms are ints.